requirements-dev.txt

```diff
@@ -1,3 +1,3 @@
-PyYAML
-pyright==1.1.257
+PyYAML==6.0
```

Comment:
I wonder if instead of repeating what's in `requirements.txt`, we could just replace this with the line `-r requirements.txt` to include that file here?

Reply:
Hmmm, that said, it seems `requirements-dev.txt` isn't used anywhere in this project. (These versions are specified in `tox.ini`).

Maybe we can move the deps from tox.ini to `requirements-dev.txt` and pin them there, then reference `-r requirements-dev.txt` in `tox.ini` instead.

Reply:
I prefer that too. I'd also like to have all deps pinned and written down in `requirements*.txt` files and have them referenced in `setup.py` or elsewhere. Is that not considered a good practice?

For eg: https://github.com/tinvaan/mazure/blob/master/setup.py#L17

Reply:
Nice -- I like your "read requirements.txt in setup.py" trick -- yeah, that sounds good to me.